asis.py

```diff
@@ -39,8 +42,7 @@
 logger.setLevel(logging.WARNING)
 
 
-class Server(object):
-    '''Regular expression for matching found headers'''
```

Comment:
Did you mean to delete this docstring? I think it was intended to apply to `headerMatch` rather than the class.